agent/tool-scripts/postprocess/GenData.pm

```diff
@@ -61,7 +61,7 @@ sub gen_data {
 			#printf "\tchart = $chart\n";
 
 			$filename = $csv . "/" . $htmlpage. "_" .$chart . ".csv";
-			open(TOOL_CSV, ">$filename") || die "$script: could not open $filename\n";
+			open(TOOL_CSV, ">>$filename") || die "$script: could not open $filename\n";
```

Comment:
Yes, this will append to it for sure.

Can we add a parameter to `gen_data()` so that the existing behavior can be preserved and only when requested we append new data?

Reply:
I think that would be fine, but make it the last option in the function call (to append), so that all other existing calls to gen_data don't have to be updated.